### DISCUSSION TITLE
Turn off Gmsh display

### DISCUSSION THREAD

**Discussion by eburkevt** [simnibs/simnibs#203] [2. General]
We are using SimNIBS to model electric fields in the brain induced by TMS and to find the optimum coil orientation. To do so, we go through 24 different angles (15 degrees per angle) and model the induced e-Field at each of these 24 orientations.  

We are using CentOS Linux v7.5.1804 and SimNIBS version 3.2.3. 

SimNIBS, after computing the e-Field each orientation, calls the Gmsh app to display graphically the induced e-field. 

Is there a way to turn off this auto display of Gmsh?  The problem is that after each orientation is calculated, we have to manually close the Gmsh app in order to finish calculating the remaining orientations.  I don't see an optional argument to suppress this graphical output.

Thanks for your help.

**Comment by axthi** [comment]
Hi,
when you script SimNIBS, please have a look at the "open_in_gmsh" parameter:
https://simnibs.github.io/simnibs/build/html/documentation/sim_struct/session.html

When using the GUI, open the Simulation Options and deactivate it there.

Best regards,
Axel
